packages/splits-sdk/src/subgraph/types.ts

```diff
@@ -678,7 +678,7 @@ export type IWaterfallModule = IAccountSharedFields & {
 
 export type IHolder = {
   address: Address
-  ownership: number
+  ownership: bigint
```

Comment:
feel pretty good we updated everywhere that uses this? and the irecipient?

Reply:
yeah, linter not throwing anything either